cmd/snapshot-create.go

```diff
@@ -0,0 +1,161 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+	"time"
+
+	"github.com/Layr-Labs/sidecar/internal/logger"
+	"github.com/spf13/cobra"
+)
+
+var (
+	snapshotCreateToPath string
+	snapshotFidelity     string
+	s3Path               string
+	compressSnapshot     bool
```

Comment:
This should likely be more specific, or offer a flag to the tune of `--compression-type=<gzip, zip, etc>`. Omitting leaves it uncompressed.